app/controllers/wishlists_for_guests_controller.rb

```diff
@@ -0,0 +1,10 @@
+class WishlistsForGuestsController < ApplicationController
+  def show
+    existing_wishlist = Wishlist.find_by(token: params[:token])
+      if existing_wishlist
```

Comment:
Sure, will do. :)
I could also use just:
`@wishlist = Wishlist.find_by!(token: params[:token])`
but that returns RecordNotFound error when no wishlist is found, so I decided to just redirect. But maybe others think otherwise?